cleanlab/filter.py

```diff
@@ -53,18 +54,18 @@
 
 
 def find_label_issues(
-    labels,
-    pred_probs,
+    labels: np.ndarray,
+    pred_probs: Optional[np.ndarray],
```

Comment:
We use `Optional` as a shorthand for `Union[None, ...]` types we provide `None` as a default argument.

In this case, I wouldn't expect users to run:

```python
from cleanlab.filter import find_label_issues
import numpy as np

labels = np.random.randint(0,2, (10,))
find_label_issues(labels, None)
```
as this doesn't pass the data-validation check at the start of the function.
Here, it's better to keep `pred_probs` as a positional argument with no default value.



```suggestion
    pred_probs: np.ndarray,
```

Reply:
Note that either annotation to `pred_probs` leads to a mypy-error:

```
cleanlab/filter.py:292: error: No overload variant of "RawArray" matches argument types "str", "ndarray[Any, Any]"  [call-overload]
```
at 
https://github.com/ChinoCodeDemon/cleanlab/blob/63400f926a5aac4308ffc22ad5f7cae1ed2e68c1/cleanlab/filter.py#L291

For now, try adding to the relevant line:

```diff
-            _pred_probs = RawArray("f", pred_probs.flatten())
+            _pred_probs = RawArray("f", pred_probs.flatten())  # type: ignore
```